src/components/form/port-input.jsx

```diff
@@ -37,7 +37,8 @@ class PortInput extends React.PureComponent {
         name="port"
         placeholder="27017"
         changeHandler={this.onPortChanged.bind(this)}
-        value={this.getPort()} />
+        value={this.getPort()}
+        type="number" />
```

Comment:
Probably not a big deal to have this completely unbound but what is the cons of re-adding here `min="0"` and `max="65535"`?

Reply:
Sure, done!